roles/docker-compose-service/README.md

```diff
@@ -0,0 +1,25 @@
+docker-service
+--------------
+
+This role configures docker-compose based services on each node
+
+The role is driven by a list of services defined in a list called `services` where each item is a dict which describes various service related configurations.
+
+
+The service list can be defined in the `hostvars` or `group_vars` ansible directory structure
+
+```yaml
+#sample service metadata
+
+services:
```

Comment:
Can I be an ass and ask for a better key name that `services` something so we know it's docker-compose

Reply:
docker-compose-services

Reply:
services-from-docker-compose